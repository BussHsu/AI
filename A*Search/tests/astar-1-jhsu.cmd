--energy 487 --start-x=2 --start-y=1 --end-x=4 --end-y=4
