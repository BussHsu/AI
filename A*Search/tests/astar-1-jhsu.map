30 25 96 95 94 93
35 20 97 98 99 92
40 15 22 33 88 91
45 10 11 44 77 90
50  5  0 55 66 85
55 60 65 70 75 80
