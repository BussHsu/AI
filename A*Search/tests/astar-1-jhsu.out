Solution steps: ['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W', 'W', 'S', 'E', 'E']
Solution cost: 486
Number of states considered: 35

Frontier:
Pos=(4, 2) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'W'] Cost=482

Closed List:
Pos=(2, 1) Moves=[] Cost=0
Pos=(1, 1) Moves=['W'] Cost=26
Pos=(1, 2) Moves=['W', 'N'] Cost=52
Pos=(2, 2) Moves=['W', 'N', 'E'] Cost=54
Pos=(1, 3) Moves=['W', 'N', 'N'] Cost=78
Pos=(1, 4) Moves=['W', 'N', 'N', 'N'] Cost=104
Pos=(2, 3) Moves=['W', 'N', 'N', 'E'] Cost=128
Pos=(1, 5) Moves=['W', 'N', 'N', 'N', 'N'] Cost=130
Pos=(0, 5) Moves=['W', 'N', 'N', 'N', 'N', 'W'] Cost=156
Pos=(0, 4) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S'] Cost=182
Pos=(0, 3) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S'] Cost=208
Pos=(0, 2) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S'] Cost=234
Pos=(0, 1) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S'] Cost=260
Pos=(3, 3) Moves=['W', 'N', 'N', 'E', 'E'] Cost=250
Pos=(0, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S'] Cost=286
Pos=(1, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E'] Cost=312
Pos=(2, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E'] Cost=338
Pos=(3, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E'] Cost=364
Pos=(4, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E'] Cost=390
Pos=(3, 1) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'N'] Cost=380
Pos=(3, 2) Moves=['W', 'N', 'N', 'E', 'E', 'S'] Cost=372
Pos=(4, 1) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'N'] Cost=400
Pos=(5, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E'] Cost=416
Pos=(5, 1) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=442
Pos=(5, 2) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N'] Cost=468
Pos=(5, 3) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N'] Cost=470
Pos=(5, 4) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N'] Cost=472
Pos=(5, 5) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N'] Cost=474
Pos=(4, 5) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W'] Cost=476
Pos=(3, 5) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W'] Cost=478
Pos=(4, 3) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'W'] Cost=474
Pos=(2, 5) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W', 'W'] Cost=480
Pos=(2, 4) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W', 'W', 'S'] Cost=482
Pos=(3, 4) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W', 'W', 'S', 'E'] Cost=484
Pos=(4, 4) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W', 'W', 'S', 'E', 'E'] Cost=486
